I think the tests are failing because the delete action is specified twice leading to a db error
```
insert into SAM_RESOURCE_TYPE_ACTION_PATTERN (resource_type_id, action_pattern, description, is_auth_domain_constrainable) values (16, 'view_status', 'view status of a billing-project', false), (16, 'stop_start_notebook_cluster', 'stop and start notebook clusters in this billing-project', false), (16, 'launch_notebook_cluster', 'launch a new notebook cluster in this billing-project', false), (16, 'alter_google_role', 'modify the role of users on the google project (only a fixed set of roles are permitted)', false), (16, 'add_to_service_perimeter', 'add this billing-project to a service-perimeter', false), (16, 'delete', '', false), (16, 'read_policy"::.+"', 'view the membership of the specified policy', false), (16, 'create_workspace', 'create new workspaces in this billing-project', false), (16, 'launch_batch_compute', 'launch workflows in this billing-project', false), (16, 'delete_notebook_cluster', '', false), (16, 'list_notebook_cluster', 'list all notebook clusters in this billing-project', false), (16, 'delete', 'delete this billing-project', false), (16, 'sync_notebook_cluster', '', false), (16, 'share_policy"::.+"', 'modify the membership of the specified policy', false), (16, 'alter_policies', 'create and delete policies for this billing-project', false), (16, 'read_policies', 'list all policies and policy details for this billing-project', false) on conflict (resource_type_id, action_pattern) do update set description = EXCLUDED.description, is_auth_domain_constrainable = EXCLUDED.is_auth_domain_constrainable

[ERROR] [17:53:39.315] [scala-execution-context-global-72] o.b.dsde.workbench.sam.Boot$ - FATAL - failure starting http server
org.postgresql.util.PSQLException: ERROR: ON CONFLICT DO UPDATE command cannot affect row a second time
```

probably should change the code to deduplicate

--- a/src/main/resources/reference.conf
+++ b/src/main/resources/reference.conf
@@ -269,7 +269,7 @@ resourceTypes = {
         description = "add this billing-project to a service-perimeter"
       }
       delete = {
-        description = "delete this billing-project"
+        description = ""
       }
     }
     ownerRoleName = "owner"

